src/localize/languages/pt-PT.json

```
@@ -0,0 +1,470 @@
+        "all_cameras": "Mostrar eventos para todas as câmeras nesta câmera",
+        "cameras": "Mostrar eventos para câmeras específicas nesta câmera",
+        "editor_label": "Opções de dependência"
+      },
+      "engines": {
+        "editor_label": "Editor de rotulos"
+      },
+      "frigate": {
+        "camera_name": "Nome da câmera do Frigate (detectado automaticamente pela entidade)",
+        "client_id": "ID do cliente do Frigate (para >1 servidor Frigate)",
+        "editor_label": "Opções do Frigate",
+        "labels": "Rotulos",
+        "url": "URL do servidor Frigate",
+        "zones": "Zonas"
+      },
+      "go2rtc": {
+        "editor_label": "Editor de rotulos",
+        "modes": {
+          "editor_label": "Editor de rotulos",
+          "mjpeg": "Mjpeg",
+          "mp4": "Mp4",
+          "mse": "Mse",
+          "webrtc": "Webrtc"
+        },
+        "stream": "Stream"
+      },
+      "hide": "Esconder",
+      "icon": "Ícone para esta câmera (detectado automaticamente pela entidade)",
+      "id": "ID exclusivo para esta câmera nesse cartão",
+      "image": {
+        "editor_label": "Editor etiquetas",
+        "refresh_seconds": "Atualizar em segundos",
+        "url": "Link"
+      },
+      "live_provider": "Fonte de visualização ao vivo para esta câmera",
+      "live_provider_options": {
+        "editor_label": "Editor de etiquetas"
+      },
+      "live_providers": {
+        "auto": "Automatico",
+        "go2rtc": "Go2rtc",
+        "ha": "Ha",
+        "image": "Imagem",
+        "jsmpeg": "JSMpeg",
+        "webrtc-card": "Cartão WebRTC (de @AlexxIT)"
+      },
+      "motioneye": {
+        "editor_label": "Directoria pre-definido",
+        "images": {
+          "directory_pattern": "Directoria pre-definido",
+          "file_pattern": "Ficheiro pre-definido"
+        },
+        "movies": {
+          "directory_pattern": "Directoria pre-definida",
+          "file_pattern": "Ficheiro pre-definido"
+        },
+        "url": "Link"
+      },
+      "title": "Título para esta câmera (detectado automaticamente pela entidade)",
+      "triggers": {
+        "editor_label": "Opções de activação",
+        "entities": "Activar a partir de outras entidades",
+        "motion": "Activar detectando automaticamente o sensor de movimento",
+        "occupancy": "Activar detectando automaticamente o sensor de ocupação"
+      },
+      "webrtc_card": {
+        "editor_label": "Opções do cartão WebRTC",
+        "entity": "Entidade de câmera de cartão WebRTC (não é uma câmera Frigate)",
+        "url": "URL da câmera do cartão WebRTC"
+      }
+    },
+    "common": {
+      "controls": {
+        "filter": {
+          "editor_label": "Editor de titulos",
+          "mode": "Modo",
+          "modes": {
+            "left": "Esquerda",
+            "none": "Nenhum",
+            "right": "Direita"
+          }
+        },
+        "next_previous": {
+          "editor_label": "Editor de rótulos",
```

coherence in translation:
```suggestion
          "editor_label": "Editor de titulos",
```